site/database/database.php

```diff
@@ -12,11 +12,7 @@ function db_init() {
    global $gl_db_init;
    global $gl_db_system;
    if (!$gl_db_init) {
-      if (co_get_system() === CO_SYSTEM_PUBLIC) {
```

Comment:
since co_get_system is no longer used, the corresponding include can be removed at the top.